pkg/libsf/session.go

```diff
@@ -30,3 +35,41 @@ func (s Session) AccessExpired() bool {
 func (s Session) RefreshExpired() bool {
 	return !s.Defined() || time.Now().After(s.RefreshExpiration)
 }
+
+type sessionTime time.Time
+
+func (t *sessionTime) UnmarshalJSON(s []byte) error {
+	r := strings.Replace(string(s), `"`, ``, -1)
```

Comment:
We can use `strings.ReplaceAll` which was created to replace that pattern and make the code more readable.